www/apps/resources/app/commerce-modules/auth/auth-flows/page.mdx

```diff
@@ -77,46 +91,63 @@ Check out the [AuthIdentity](/references/auth/models/AuthIdentity) reference for
 
 ## Auth Flow 2: Third-Party Service Authentication
 
-The third-party service authentication method requires using the `authenticate` method first.
-
-If the `authenticate` method returns the following object:
+The third-party service authentication method requires using the `authenticate` method first:
 
 ```ts
-data = {
-  success: true,
-  location: "https://....",
+const { success, authIdentity, location } = await authModuleService.authenticate(
+  "emailpass",
+  // passed to auth provider
+  {
+    // ...
+  }
+)
+
+if (location) {
+  // return the location for the front-end to redirect to
+}
+
+if (!success) {
+  // authentication failed
 }
+
+// authentication successful
 ```
 
-It means the authentication process requires the user to perform an action with a third-party service. For example, when using the `google` provider, the user goes to the URL specified in the `location`'s value to log in with their Google account. 
+If the `authenticate` method returns a `location` property, the authentication process requires the user to perform an action with a third-party service. So, you return the `location` to the front-end or client to redirect to that URL.
+
+For example, when using the `google` provider, the `location` is the URL that the user is navigated to login.
+
+If the `authenticate` method doesn't return a `location` property and the returned `success` property, the user is successfully authenticated. This happens after the user logs-in with the provider, such as google, the first time.
```

Comment:
In the `authenticate` flow we always return a `location`, there is no way to have the user authenticated successfully at this point. That has to happen after calling the `/callback` route / sdk method

Reply:
Ah I forgot to update this part! Will fix it